<launch>  
    <!--<node pkg="practice" type="vel2odom.py" name="vel2odom"/>-->
    <node pkg="robot_localization" type="ekf_localization_node" name="ekf_localization0">
        <rosparam command="load" file="$(find practice)/config/ekf_localization0.yaml"/>
        <remap from="odometry/filtered" to="/odometry/filtered_mid" />
    </node>
    <!--
    <node pkg="robot_localization" type="navsat_transform_node" name="navsat_transform" output="screen">
        <rosparam command="load" file="$(find practice)/config/navsat_transform.yaml"/>
        <remap from="odometry/filtered" to="/odometry/filtered_mid" />
      <param name="publish_filtered_gps" value="true" />
    </node>
        
    <node pkg="robot_localization" type="ekf_localization_node" name="ekf_localization1">
        <rosparam command="load" file="$(find practice)/config/ekf_localization1.yaml"/>
        <remap from="odometry/filtered" to="/odometry/filtered" />
    </node> 
    -->
</launch>
